domain/model/modelmigration/export_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package modelmigration
+
+import (
+	"context"
+
+	"github.com/juju/description/v8"
+	gomock "go.uber.org/mock/gomock"
+	gc "gopkg.in/check.v1"
+
+	coremodel "github.com/juju/juju/core/model"
+	"github.com/juju/juju/core/model/testing"
+)
+
+type exportSuite struct {
+	modelExportService *MockExportService
+}
+
+var _ = gc.Suite(&exportSuite{})
+
+func (e *exportSuite) setupMocks(c *gc.C) *gomock.Controller {
+	ctrl := gomock.NewController(c)
+
+	e.modelExportService = NewMockExportService(ctrl)
+
+	return ctrl
+}
+
+func (e *exportSuite) TestModelEnvironVersionExport(c *gc.C) {
+	defer e.setupMocks(c).Finish()
+
+	newUUID := testing.GenModelUUID(c)
+	model := description.NewModel(description.ModelArgs{
+		EnvironVersion: 5,
+		Config: map[string]interface{}{
+			"uuid": newUUID.String(),
+		},
+	})
+	c.Check(model.Tag().Id(), gc.Equals, newUUID.String())
```

Comment:
Not sure either of these checks are providing value. We have these checks in the description package and don't need to double test here.